docs/en/rules/Azure.MariaDB.UseSSL.md

```diff
@@ -0,0 +1,33 @@
+---
+severity: Critical
+pillar: Security
+category: Data protection
+resource: Azure Database for MariaDB
+online version: https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.MariaDB.UseSSL/
+---
+
+# Encrypted connections
+
+## SYNOPSIS
+
+Azure Database for MariaDB servers should only accept encrypted connections.
+
+## DESCRIPTION
+
+Azure Database for MariaDB is configured to only accept encrypted connections by default.
+When the setting _enforce SSL connections_ is disabled, encrypted and unencrypted connections are permitted.
+This does not indicate that unencrypted connections are being used.
+
+Unencrypted communication to MariaDB server instances could allow disclosure of information to an untrusted party.
+
+## RECOMMENDATION
+
+Azure Database for MariaDB should be configured to only accept encrypted connections.
+Unless explicitly required, consider enabling _enforce SSL connections_.
+
+Also consider using Azure Policy to audit or enforce this configuration.
+
```

Comment:
We need some code examples for this ARM and Bicep.

Reply:
Yes, I'll fix that. I took another `UseSSL` rule as reference. I'll update that as well later on with some ARM and Bicep examples.